src/layouts/components/Header.tsx

```diff
@@ -34,103 +39,167 @@ export function Header(): JSX.Element {
+                        role="button"
+                        className="h-6 w-6 text-primary-500"
+                        onClick={() => setMenu(true)}
+                        data-testid="Header.OpenMenu"
+                      />
+                    )}
+                  </div>
+                </>
               )}
             </div>
-          </div>
-        </Container>
-      </div>
-
-      <>{menu && <MobileMenu />}</>
-    </header>
+          </Container>
+        </div>
+      </header>
+      <>
+        {menu && (
+          <>
+            <div className="fixed z-50 md:hidden">
+              <MobileMenu toggleMenu={() => setMenu(false)} />
+            </div>
+            <div className="w-full hidden md:block md:fixed md:z-50">
+              <TabletMenu toggleMenu={() => setMenu(false)} />
+            </div>
+          </>
+        )}
+      </>
+    </>
   );
 }
 
 function DesktopNavbar(): JSX.Element {
   return (
-    <div className="hidden md:flex ml-2 lg:ml-8 md:w-full md:justify-between items-center text-gray-600 dark:text-dark-gray-900">
+    <div className="ml-2 hidden items-center text-gray-600 dark:text-dark-gray-900 md:w-full md:justify-between lg:ml-8 lg:flex">
       <div className="hidden md:flex">
         <HeaderLink
-          className="ml-1 lg:ml-4"
+          className="ml-1 lg:ml-2"
           text="DEX"
           pathname="/dex"
           testId="Desktop.HeaderLink.DEX"
         />
         <HeaderLink
-          className="ml-1 lg:ml-4"
+          className="ml-1 lg:ml-2"
           text="Blocks"
           pathname="/blocks"
           testId="Desktop.HeaderLink.Blocks"
         />
         <HeaderLink
-          className="ml-1 lg:ml-4"
+          className="ml-1 lg:ml-2"
           text="Vaults"
           pathname="/vaults"
           testId="Desktop.HeaderLink.Vaults"
         />
         <HeaderLink
-          className="ml-1 lg:ml-4"
+          className="ml-1 lg:ml-2"
           text="Auctions"
           pathname="/auctions"
           testId="Desktop.HeaderLink.Auctions"
         />
         <HeaderLink
-          className="ml-1 lg:ml-4"
+          className="ml-1 lg:ml-2"
           text="Oracles"
           pathname="/oracles"
           testId="Desktop.HeaderLink.Oracles"
         />
         <HeaderLink
-          className="ml-1 lg:ml-4"
+          className="ml-1 lg:ml-2"
           text="Tokens"
           pathname="/tokens"
           testId="Desktop.HeaderLink.Tokens"
         />
         <HeaderLink
-          className="ml-1 lg:ml-4"
+          className="ml-1 lg:ml-2"
           text="Masternodes"
           pathname="/masternodes"
           testId="Desktop.HeaderLink.Masternodes"
         />
+
+        {/* <HeaderLink
+          className="ml-1 lg:ml-4"
+          text="Consortium"
+          pathname="/consortium/asset_breakdown"
+          testId="Desktop.HeaderLink.Consortium"
+        /> */}
+
+        {useWindowDimensions().width <= 1530 ? (
```

Comment:
can we declare width variable outside JSX element?  hooks should get used outside JSX element

Reply:
removed check for width tab will be grouped under `More`